tokens/base/alpha.json

```diff
@@ -0,0 +1,1124 @@
+{
+    "palette": {
+        "neutral": {
+            "A25": {
+                "type": "color",
+                "value": {
+                    "colorSpace": "oklch",
+                    "components": [
+                        0.1149,
+                        0,
+                        0
+                    ],
+                    "alpha": 0.02,
+                    "hex": "#05050505"
+                }
+            },
+            "A50": {
```

Comment:
after transform token would be `base.neutralA50`. Is that expected name?

Reply:
yeah i guess A stands for alpha, we can either say `alpha40` or `Alpha50` maybe that's more clear?